index.html

```diff
@@ -32,7 +32,7 @@
     </div>
     <div id="log">
     </div>
-    <input id="command" placeholder="enter command" autofocus="true" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" />
+    <input id="command" placeholder="enter aircraft command" autofocus="true" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" />
```

Comment:
is this file even used?  our html is generated from the handlebars templates and output to the `public` dir.